proto/bid.proto

```diff
@@ -1,29 +0,0 @@
-syntax = "proto3";
```

Comment:
What's the reason to do this?

Reply:
Because of `bid.proto` imports `marketplace.proto` to re-use some types, and `marketplace.proto` imports `bid.proto` to use `BidOrder` message as parameter for `CreateOrder` method.